### PR TITLE
Retranslate merge_v1

### PR DESCRIPTION
**Pull request by Uwonsang**
번역 초벌입니다. 이후에 convention과 띄어쓰기 맞춤법을 검사할 예정입니다.  내용 어색하신거 있으시면 확인해주세요~
#138

### CODE REVIEW

File: sources/layers/merge.md [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
 ### Add
 
 ```python
 keras.layers.Add()
 ```
 
-Layer that adds a list of inputs.
+입력 리스트를 더하는 층입니다.
+
+모든 형태의 텐서 리스트를 입력으로 받아 같은 모양을 가진 하나의 텐서로 반환합니다.
```
Comment (Naruu) [inline sources/layers/merge.md]: all of the same shape : 모든 형태의 -> 모두 같은 형태의
가 더 어울리지 않을까 싶어요 :)

Comment (Naruu) [inline sources/layers/merge.md]: 입력으로 받아 ~텐서로 반환합니다.
-> 입력받아 ~텐서를 반환합니다.
가 더 자연스러울 것 같아요!

Comment (Naruu) [inline sources/layers/merge.md]: shape을 모양, 크기로 섞어서 사용하셨는데 모두 '형태'로 통일하는 게 좋을 것 같습니다 :)

File: sources/layers/merge.md [hunk 2]
```diff
@@ -27,23 +29,25 @@ added = keras.layers.Add()([x1, x2])
...
 ### Subtract
 
 ```python
 keras.layers.Subtract()
 ```
 
-Layer that subtracts two inputs.
+두 입력에 대해 뺄셈을 수행하는 층입니다.
+
+동일한 크기를 가진 텐서 리스트(2개의 텐서)를 입력으로 받아
```
Comment (Naruu) [inline sources/layers/merge.md]: 형태가 동일한(같은), 길이가 2인 텐서의 리스트를 입력받아

File: sources/layers/merge.md [hunk 3]
```diff
@@ -27,23 +29,25 @@ added = keras.layers.Add()([x1, x2])
...
+동일한 크기의 단일 텐서로 반환합니다. (inputs[0] - inputs[1])
```
Comment (Naruu) [inline sources/layers/merge.md]: 동일한 형태의 텐서(inputs[0] - inputs[1]) 1개를 반환합니다.

File: sources/layers/merge.md [hunk 4]
```diff
@@ -149,23 +150,16 @@ __Arguments__
 keras.layers.Dot(axes, normalize=False)
 ```
 
-Layer that computes a dot product between samples in two tensors.
```
Comment (Naruu) [inline sources/layers/merge.md]: 두 텐서의 샘플들 간에 내적 연산을 수행하는 층입니다.
샘플 앞에 두 개라는 단어가 들어갈 필요가 없을 것 같아요

File: sources/layers/merge.md [hunk 5]
```diff
@@ -149,23 +150,16 @@ __Arguments__
...
 
-- __axes__: Integer or tuple of integers,
-    axis or axes along which to take the dot product.
-- __normalize__: Whether to L2-normalize samples along the
-    dot product axis before taking the dot product.
-    If set to True, then the output of the dot product
-    is the cosine proximity between the two samples.
-- __**kwargs__: Standard layer keyword arguments.
-    
+- __axes__:  단일 정수이거나  정수 튜플이어야 하며, 내적 연산을 수행할 축 또는 축들을 의미합니다.
```
Comment (Naruu) [inline sources/layers/merge.md]: 한국어에서는 복수형이어도 '~들'이라는 표현을 잘 사용하지 않는다고 해요
'내적 연산을 수행할 축을 의미합니다'로 적는 것이 더 자연스러울 것 같아요

File: sources/layers/merge.md [hunk 6]
```diff
@@ -175,19 +169,20 @@ __Arguments__
...
 
+`Add` 층에 대해 함수처럼 제공되는 인터페이스 입니다.
+
+__인수__
 
-Functional interface to the `Add` layer.
+- __inputs__: 입력 텐서에 대한 리스트입니다.(최소 2개 이상의 텐서를 포함해야합니다).
 
-__Arguments__
+- __**kwargs__: 표준 레이어 키워드 인자입니다.
```
Comment (Naruu) [inline sources/layers/merge.md]: 레이어 -> 층

File: sources/layers/merge.md [hunk 7]
```diff
@@ -175,19 +169,20 @@ __Arguments__
...
 
-- __inputs__: A list of input tensors (at least 2).
-- __**kwargs__: Standard layer keyword arguments.
 
-__Returns__
+__반환값__
 
-A tensor, the sum of the inputs.
```
Comment (Naruu) [inline sources/layers/merge.md]: sum은 전체 값을 더한 총합을 의미하니,
입력값의 합인 텐서
가 좀 더 명확할 것 같아요:)

File: sources/layers/merge.md [hunk 8]
```diff
@@ -212,19 +207,18 @@ model = keras.models.Model(inputs=[input1, input2], outputs=out)
...
-__Arguments__
+- __inputs__: 입력 텐서에 대한 리스트입니다. (정확히 2개의 텐서이어야 합니다.)
+- __**kwargs__: 표준 레이어 키워드 인자입니다.
 
-- __inputs__: A list of input tensors (exactly 2).
-- __**kwargs__: Standard layer keyword arguments.
+__반환값__
 
-__Returns__
+입력되는 두 개의 텐서에 대한 차이값이 단일 텐서로 반환됩니다.
```
Comment (Naruu) [inline sources/layers/merge.md]: 두 입력값의 차를 단일 텐서(1개의 텐서)로 반환합니다.

File: sources/layers/merge.md [hunk 9]
```diff
@@ -212,19 +207,18 @@ model = keras.models.Model(inputs=[input1, input2], outputs=out)
 keras.layers.subtract(inputs)
 ```
 
+`Subtract` 층에 대해 함수처럼 제공되는 인터페이스입니다.
 
-Functional interface to the `Subtract` layer.
+__인수__
 
-__Arguments__
+- __inputs__: 입력 텐서에 대한 리스트입니다. (정확히 2개의 텐서이어야 합니다.)
```
Comment (Naruu) [inline sources/layers/merge.md]: 입력 텐서의 리스트
'~에 대한'이 흔히 사용하는 번역투라고 해요!

File: sources/layers/merge.md [hunk 10]
```diff
@@ -291,17 +283,16 @@ A tensor, the average of the inputs.
...
-- __inputs__: A list of input tensors (at least 2).
-- __**kwargs__: Standard layer keyword arguments.
+- __inputs__: 입력 텐서에 대한 리스트입니다. 최소 2개 이상의 텐서를 포함해야 합니다.
+- __**kwargs__: 표준 층 키워드 인자입니다.
 
-__Returns__
+__반환값__
 
-A tensor, the element-wise maximum of the inputs.
+입력 텐서들에 대해 입력 단위로 최댓값 연산을 수행된 결과가 단일 텐서로 반환됩니다.
```
Comment (Naruu) [inline sources/layers/merge.md]: element-wise는 요소별 연산로 번역하는 것이 더 어울릴 것 같아요.
입력값의 요소별로 최댓값 연산을 수행한 결과를 담은 텐서
좀 더 나은 표현이 무엇이 있을까요 🤔

File: sources/layers/merge.md [hunk 11]
```diff
@@ -1,18 +1,18 @@
...
 ### Add
 
 ```python
 keras.layers.Add()
 ```
 
-Layer that adds a list of inputs.
+입력 리스트를 더하는 층<sub>Layer</sub>입니다.
+
+모든 형태의 텐서 리스트를 입력받아 동일한 형태를 가진 하나의 텐서로 반환합니다.
```
Comment (Naruu) [inline sources/layers/merge.md]: 동일한 형태를 가진 -> 동일한 형태의/같은 형태
이 더 자연스러울 것 같아요~

File: sources/layers/merge.md [hunk 12]
```diff
@@ -27,24 +27,25 @@ added = keras.layers.Add()([x1, x2])
...
 <span style="float:right;">[[source]](https://github.com/keras-team/keras/blob/master/keras/layers/merge.py#L231)</span>
+
 ### Subtract
 
 ```python
 keras.layers.Subtract()
 ```
 
-Layer that subtracts two inputs.
+두 입력에 대해 뺄셈을 하는 층입니다.
```
Comment (Naruu) [inline sources/layers/merge.md]: add는 더하는 층이라고 되어있는데, subtract는 뺄셈을 하는 층이라고 되어있네요
덧셈, 뺄셈이나 더하는 층과 빼는 층으로 통일하는 것이 좋을 것 같아요!

File: sources/layers/merge.md [hunk 13]
```diff
@@ -27,24 +27,25 @@ added = keras.layers.Add()([x1, x2])
...
+
+동일한 형태를 가진 텐서 리스트(2개의 텐서)를 입력으로 받아
```
Comment (Naruu) [inline sources/layers/merge.md]: 동일한 형태를 가진 -> 동일한 형태의/ 같은 형태의
가 자연스러울 것 같습니다

File: sources/layers/merge.md [hunk 14]
```diff
@@ -59,136 +60,125 @@ subtracted = keras.layers.Subtract()([x1, x2])
...
 keras.layers.Multiply()
 ```
 
-Layer that multiplies (element-wise) a list of inputs.
+입력 리스트를 요소 단위<sub>Element-wise</sub>로 곱하는 층입니다.
 
-It takes as input a list of tensors,
-all of the same shape, and returns
-a single tensor (also of the same shape).
+모튼 형태의 텐서 리스트를 입력으로 받아 동일한 형태를 가진 하나의 텐서로 반환합니다.
```
Comment (Naruu) [inline sources/layers/merge.md]: 오타가 있네요
모튼 -> 모든

File: sources/layers/merge.md [hunk 15]
```diff
@@ -59,136 +60,125 @@ subtracted = keras.layers.Subtract()([x1, x2])
...
 
 <span style="float:right;">[[source]](https://github.com/keras-team/keras/blob/master/keras/layers/merge.py#L298)</span>
+
 ### Maximum
 
 ```python
 keras.layers.Maximum()
 ```
 
-Layer that computes the maximum (element-wise) a list of inputs.
```
Comment (Naruu) [inline sources/layers/merge.md]: 입력단위의 요소별로 최댓값을 계산하는 층입니다.
가 좀 더 자연스러울 것 같아요

Comment (Uwonsang) [reply sources/layers/merge.md]: 입력 리스트안에 있는 값들 중에서 최대값을 구하는 레이어인데,
제가 했던 번역은 의미가 잘 다가오지 않는 편인가요?

입력 리스트의 요소 중에서 최댓값을 계산하는 층입니다. 이건 어떠신가요 @Naruu 

괜찮으시면 이렇게 밑에 Minimum도 바꾸겠습니다.

File: sources/layers/merge.md [hunk 16]
```diff
@@ -127,7 +127,7 @@ keras.layers.Minimum()
 keras.layers.Concatenate(axis=-1)
 ```
 
-입력 리스트를 이어붙이기<sub>Concatenate</sub>를 하는 층입니다.
+입력 리스트를 이어붙이기<sub>concatenate</sub>를 하는 층입니다.
```
Comment (Naruu) [inline sources/layers/merge.md]: 입력 리스트를 이어붙이는<sub>concatenate</sub> 층입니다.
자연스럽지 않을까요?

File: sources/layers/merge.md [hunk 17]
```diff
@@ -127,7 +127,7 @@ keras.layers.Minimum()
...
 
 이어붙이기 축을 제외하고 동일한 형태의 모든 텐서 리스트를 입력으로 받아
```
Comment (Naruu) [inline sources/layers/merge.md]: 이어붙이기 축 -> 이어붙이는 축

File: sources/layers/merge.md [hunk 18]
```diff
@@ -59,136 +60,125 @@ subtracted = keras.layers.Subtract()([x1, x2])
...
 ```
 
-Layer that computes a dot product between samples in two tensors.
+두 개의 텐서에 대해 내적 연산<sub>dot product</sub>를 하는 층입니다.
 
-E.g. if applied to a list of two tensors `a` and `b` of shape `(batch_size, n)`,
-the output will be a tensor of shape `(batch_size, 1)`
-where each entry `i` will be the dot product between
-`a[i]` and `b[i]`.
+예) 만약 형태가 `(batch_size, n)`인 두 텐서 `a`와 `b`를 갖는 리스트에서 수행될 경우, 출력 텐서의 형태는 `(batch_size, 1)`가 됩니다. 출력 텐서의 각 요소 `i`는 `a[i]`와 `b[i]` 간의 내적 연산입니다.
```
Comment (Naruu) [inline sources/layers/merge.md]: 출력 텐서의 각 요소 `i`는 -> 출력텐서의 i번째 원소는 
같은 의미이면 뒷 표현이 좀 더 자연스러울 것 같아요

File: sources/layers/merge.md [hunk 19]
```diff
@@ -59,136 +60,125 @@ subtracted = keras.layers.Subtract()([x1, x2])
...
-### add
+------
 
+### add
 
 ```python
 keras.layers.add(inputs)
 ```
 
+`Add` 층에 대해 함수처럼 제공되는 인터페이스 입니다.
```
Comment (Naruu) [inline sources/layers/merge.md]: 인터페이스입니다.
붙여써주세요~

File: sources/layers/merge.md [hunk 20]
```diff
@@ -59,136 +60,125 @@ subtracted = keras.layers.Subtract()([x1, x2])
...
-Functional interface to the `Add` layer.
-
-__Arguments__
-
-- __inputs__: A list of input tensors (at least 2).
-- __**kwargs__: Standard layer keyword arguments.
+__인자__
 
-__Returns__
+- __inputs__: 입력 텐서의 리스트입니다. 최소 2개 이상의 텐서를 포함해야합니다.
```
Comment (Naruu) [inline sources/layers/merge.md]: 포함해야 합니다
띄어쓰기 해주세요~

File: sources/layers/merge.md [hunk 21]
```diff
@@ -59,136 +60,125 @@ subtracted = keras.layers.Subtract()([x1, x2])
...
+- __**kwargs__: 표준  키워드 인자입니다.
```
Comment (Naruu) [inline sources/layers/merge.md]: '표준'과 '키워드' 사이에 공백이 2개인 것 같아요

File: sources/layers/merge.md [hunk 22]
```diff
@@ -59,136 +60,125 @@ subtracted = keras.layers.Subtract()([x1, x2])
...
-    axis or axes along which to take the dot product.
-- __normalize__: Whether to L2-normalize samples along the
-    dot product axis before taking the dot product.
-    If set to True, then the output of the dot product
-    is the cosine proximity between the two samples.
-- __**kwargs__: Standard layer keyword arguments.
-    
-----
+- __axes__:  단일 정수이거나  정수 튜플이어야 하며, 내적 연산을 수행할 축을 의미합니다.
+- __normalize__: 내적 연산을 수행하기 전에 샘플들에 L2 정규화<sub>l2-normalize</sub>를 수행 여부입니다. True로 설정하면, 내적 연산의 결과는 두 샘플 간의 코싸인 유사도와 같습니다.
```
Comment (Naruu) [inline sources/layers/merge.md]: 수행 여부입니다.
->를 수행할지 결정합니다.
는 어떠신가요?

File: sources/layers/merge.md [hunk 23]
```diff
@@ -239,137 +226,125 @@ subtracted = keras.layers.subtract([x1, x2])
...
-- __inputs__: A list of input tensors (at least 2).
-- __**kwargs__: Standard layer keyword arguments.
+- __inputs__: 입력 텐서의 리스트입니다. 최소 2개 이상의 텐서를 포함해야 합니다.
+- __**kwargs__: 표준 층 키워드 인자입니다.
 
-__Returns__
+__반환값__
 
-A tensor, the average of the inputs.
+입력 텐서들의 평균값을 연산 결과가 단일 텐서로 반환됩니다.
```
Comment (Naruu) [inline sources/layers/merge.md]: 입력 텐서들의 평균값을 단일 텐서로 반환합니다.

File: sources/layers/merge.md [hunk 24]
```diff
@@ -239,137 +226,125 @@ subtracted = keras.layers.subtract([x1, x2])
...
-- __inputs__: A list of input tensors (at least 2).
-- __**kwargs__: Standard layer keyword arguments.
+- __inputs__: 입력 텐서의 리스트입니다. 최소 2개 이상의 텐서를 포함해야 합니다.
+- __**kwargs__: 표준 층 키워드 인자입니다.
 
-__Returns__
+__반환값__
 
-A tensor, the element-wise minimum of the inputs.
+입력 텐서들에 대해 입력 단위로 최댓값 연산을 수행된 결과가 단일 텐서로 반환됩니다.
```
Comment (Naruu) [inline sources/layers/merge.md]: element-wise가 입력 단위로 번역된 것 같습니다.

File: sources/layers/merge.md [hunk 25]
```diff
@@ -239,137 +226,125 @@ subtracted = keras.layers.subtract([x1, x2])
...
-- __inputs__: A list of input tensors (at least 2).
-- __axes__: Integer or tuple of integers,
-    axis or axes along which to take the dot product.
-- __normalize__: Whether to L2-normalize samples along the
-    dot product axis before taking the dot product.
-    If set to True, then the output of the dot product
-    is the cosine proximity between the two samples.
-- __**kwargs__: Standard layer keyword arguments.
+- __inputs__: 입력 텐서의 리스트입니다. 최소 2개 이상의 텐서를 포함해야 합니다.
+- __axes__: 단일 정수거나 튜플 형태의 정수여야 하며, 내적 연산을 수행할 축 또는 축들을 의미합니다.
```
Comment (Naruu) [inline sources/layers/merge.md]: 한국어에서는 '축들'처럼 복수 표현을 잘 사용하지 않는 것으로 알고 있어요.
'수행할 축'이라고만 적거나 '수행할 축 여러 개' 정도로 표현하는 건 어떨까요?

**Comment by Uwonsang** [comment]
> 정수
> 처럼 자료형을 나타내는 단어들은 `int`처럼 코드 블럭으로 표시해주세요!

**Comment by Uwonsang** [comment]
정수의 경우 int 가 아닌 interger로 표기 되어잇어서 그대로 정수라고 적었습니다.

**Comment by fuzzythecat** [comment]
감사합니다!
